pkg/idtools/idtools.go

```diff
@@ -165,35 +159,37 @@ func (i *IdentityMapping) ToContainer(pair Identity) (int, int, error) {
 	if i == nil {
 		return pair.UID, pair.GID, nil
 	}
-	uid, err := toContainer(pair.UID, i.uids)
+	uid, err := toContainer(pair.UID, i.UIDMap)
 	if err != nil {
 		return -1, -1, err
 	}
-	gid, err := toContainer(pair.GID, i.gids)
+	gid, err := toContainer(pair.GID, i.GIDMap)
 	return uid, gid, err
 }
 
 // Empty returns true if there are no id mappings
 func (i *IdentityMapping) Empty() bool {
-	return i == nil || (len(i.uids) == 0 && len(i.gids) == 0)
+	return i == nil || (len(i.UIDMap) == 0 && len(i.GIDMap) == 0)
 }
 
-// UIDs return the UID mapping
-// TODO: remove this once everything has been refactored to use pairs
+// UIDs returns the mapping for UID.
+//
+// Deprecated: use the UIDMap field directly.
```

Comment:
Slightly wondering if this would still be useful / convenient to have (to handle the nil cases), but I'd have to check again where / how it's all used (otherwise, good with me to clean it up 👍)

Reply:
Oh yeah, the nil cases... 🤦 I made a mistake. There's at least one case where an `*IdentityMapping` function argument is immediately dereferenced without any nil-pointer checks. That's no good; excuse me while I go revise this PR.

Reply:
Oh! Good that I commented then (probably shouldn't have "LGTM'd while on the road 🤦); no worries!